docs/bot-optimization.md

```diff
@@ -3,78 +3,97 @@ This page explains where to customize your strategies, and add new
 indicators. 
 
 ## Table of Contents
-- [Change your strategy](#change-your-strategy)
+- [Install a custom strategy file](#install-a-custom-strategy-file)
+- [Customize your strategy](#change-your-strategy)
 - [Add more Indicator](#add-more-indicator)
+- [Where is the default strategy](#where-is-the-default-strategy)
+
+Since the version `0.16.0` the bot allows using custom strategy file.
+
+## Install a custom strategy file
+This is very simple. Copy paste your strategy file into the folder 
+`user_data/strategies`.
+
+Let guess you have a strategy file `awesome-strategy.py`:
```

Comment:
"assume" instead of "guess"?

Reply:
Thank you very much for fixing my English.
This is fixed.